upcloud/request/server_test.go

```diff
@@ -108,64 +109,83 @@ func TestCreateServerRequest(t *testing.T) {
 
 	expectedJSON := `
 	{
-      "server": {
```

Comment:
The only actual modification here is adding `"server_group": "test-group"` line to this json. Github shows it as rewritten because I also formatted it